docs/design/README.md

```diff
@@ -582,3 +585,21 @@ This will require passing all `-e BUILDSYS_SOMETHING=foo` variables to the `twol
 Getting the Bottlerocket build cut-over to using `twoliter make` is the first milestone of the Twoliter project.
 
 `*` Another option is to use rewrite Bottlerocket's `Makefile.toml` to explicitly call `twoliter make` in each of its targets.
+
+## Twoliter Alpha
+
+Once Bottlerocket's build system has been moved to Twoliter, we want to introduce the ability to create variants sooner rather than later.
+The introduction of kits into the build is complicated and needs to be done thoughtfully.
+Rather than rush the development of kits, we want to introduce a `twoliter build variant` command that does not require kits.
+We call this Twoliter Alpha, and it can be thought of as a minimum viable product.
+Ideally the interface of the `twoliter build variant` command will not change much once kits are introduced.
+Project structure will change once kits are introduced and backward compatibility with Twoliter Alpha is not likely.
+Thus, Alpha should be used for development and experimentation, not production.
```

Comment:
Do you expect an incompatibility beyond that `Twoliter.toml` will need to be updated to reflect the kit dependencies, and to remove the SDK & toolchain image overrides?

That seems like a decent compatibility story - it's a breaking change but shouldn't affect project layout, unless I'm missing something.

Reply:
Project layout will include a new directory and Cargo workspace named Kits. Cargo.toml metadata structures may change. I'm sure backward compatibility is not entirely impossible, but it may be more challenging than it's worth.